Invalid web-sys minimal version

**Problem**

The minimal version of `web-sys` is `0.3.56` (package/web/Cargo.toml) and the `package/web/src/events/mounted.rs` module use the `set_behavior` method that is existing since `0.3.70`.


That's trigger compilation issues when we build the project with minimal version.

```
error[E0599]: no method named `set_behavior` found for struct `ScrollIntoViewOptions` in the current scope
  --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/dioxus-web-0.6.0-alpha.5/src/events/mounted.rs:74:25
   |
74 |                 options.set_behavior(web_sys::ScrollBehavior::Instant);
   |                         ^^^^^^^^^^^^
   |
help: there is a method `behavior` with a similar name
   |
74 |                 options.behavior(web_sys::ScrollBehavior::Instant);
   |                         ~~~~~~~~
```

**Environment:**
 - Dioxus version:  0.6.0-alpha.5
 - Rust version:    1.82.0
 - OS info:         MacOS
 - App platform:    web`

**Questionnaire**

I would like to fix and I have a solution (bump web-sys to 0.3.70)

--- a/packages/web/Cargo.toml
+++ b/packages/web/Cargo.toml
@@ -24,7 +24,7 @@ dioxus-interpreter-js = { workspace = true, features = [
 ] }
 generational-box = { workspace = true }
 
-js-sys = "0.3.56"
+js-sys = "0.3.70"
 wasm-bindgen = { workspace = true }
 wasm-bindgen-futures = "0.4.29"
 tracing = { workspace = true }
@@ -44,7 +44,7 @@ ciborium = { workspace = true, optional = true }
 async-trait = { version = "0.1.58", optional = true }
 
 [dependencies.web-sys]
-version = "0.3.56"
+version = "0.3.70"
 features = [
     "AnimationEvent",
     "ClipboardEvent",

